Expensify/App
# fix Locale Picker and small picker styles

### Details

1. Removed API call.
2. Fixed the Picker style so that it shows a blue border while focused.
3. Fixed picker style on Firefox browsers. https://expensify.slack.com/archives/C01GTK53T8Q/p1629962984011400


### Fixed Issues

$ https://github.com/Expensify/App/issues/4513

### Tests | QA Steps
1. Open the login page & sign in with some account.
2. Sign out of the app.
3. Switch a language.
4. Language should change.

### Tested On


### Screenshots


#### Web | Desktop

![locale-wd](https://user-images.githubusercontent.com/24370807/130902477-2ad6843f-80ea-4790-a479-185c122fd8fc.png)

#### Mobile Web

![locale-m](https://user-images.githubusercontent.com/24370807/130903587-994b0bf6-dbec-47bd-80a0-82225f3b1ecf.png)

#### iOS


#### Android

![locale-a](https://user-images.githubusercontent.com/24370807/130903576-4b8a38f7-10df-475f-bd57-703b31d56709.png)
